.github/workflows/ci.yml

```diff
@@ -18,13 +18,13 @@ jobs:
       with:
         name: Maven
         path: ~/.m2/
-      
+
   test_examples:
     needs: build
     runs-on: ubuntu-latest
     strategy:
      matrix:
-        language: [cpp, java, jni, kotlin]
+        language: [cpp, java] #, jni, kotlin]
```

Comment:
Are we permanently removing JNI and Kotlin or just temporarily?

Reply:
Temporarily.